### DISCUSSION TITLE
Do the systems that support sending gameid to BlueRetro require an internal installation?

### DISCUSSION THREAD

**Discussion by Koohiisan** [darthcloud/BlueRetro#1019] [General]
Or will it work somehow just being plugged in to the controller ports on those systems externally?  Specifically interested in GameCube, but I read in another discussion that Krikzz's ED64 might also do gameid.  Would that need to be internal too?  Just wondering how the gameid gets to the device, I suppose.

**Comment by darthcloud** [comment]
Yes PS1, GC & N64 gameid work on any type of install.
